vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -375,29 +381,32 @@ func (cluster *ClusterState) garbageCollectAggregateCollectionStates(now time.Ti
 // RateLimitedGarbageCollectAggregateCollectionStates removes obsolete AggregateCollectionStates from the ClusterState.
 // It performs clean up only if more than `gcInterval` passed since the last time it performed a clean up.
 // AggregateCollectionState is obsolete in following situations:
-// 1) It has no samples and there are no more active pods that can contribute,
+// 1) It has no samples and there are no more contributive pods that can contribute,
```

Comment:
"no more contributive pods that can contribute" sounds a bit redundant.

"no more contributive pods (explanation what is a contributive pod)"

?

Reply:
Refactored, thanks.